pkg/demo/explainer/Library/2-SimpleDecorator.js

```diff
@@ -0,0 +1,101 @@
+/**
+ * @license
+ * Copyright 2021 Google LLC.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+/* globals log, html */
+({
+  render({fonts}) {
+    return {
+      fonts: {
+        models: fonts,
+        $template: 'font_t',
+        decorator: 'decorator'
+      }
+    };
+  },
+
+  decorator({family, fullName, weight, style}) {
+    const fweight = style.includes('Bold') ? 'bold' : weight;
```

Comment:
This helper logic is common, maybe worth moving to some shared util at some point? I'm not sure what the right pattern is for that in this Arcs impl.

Reply:
I don't think you can 'import' in non-permissive particles? Maybe Scott can provide a helper function for dynamic imports, but I think you'd need to "await" it.

Reply:
With`Runtime.particleOptions` you can inject whatever API you like into Particle-scope:
```
// customize particle scope
Runtime.particleOptions = {
  injections: {
    // items to make available in each Particle's module scope
    fetch: fetch.bind(window)
  }
};
```